src/id/doujindesu/src/eu/kanade/tachiyomi/extension/id/doujindesu/DoujinDesu.kt

```diff
@@ -414,13 +472,97 @@ class DoujinDesu : ParsedHttpSource(), ConfigurableSource {
             Seri             : $seriesParser
             """.trimIndent()
         } else {
-            val showDescription = infoElement.selectFirst("div.pb-2 > p:nth-child(1)")!!.text()
-            """
-            $showDescription
+            val pb2Element = infoElement.selectFirst("div.pb-2")
 
-            Judul Alternatif : $alternativeTitle
-            Seri             : $seriesParser
-            """.trimIndent()
+            val showDescription = if (pb2Element != null) {
+                val paragraphs = pb2Element.select("p")
+                val firstText = paragraphs.firstOrNull()?.text()?.trim()?.lowercase()
+
+                // CASE 1: Manga 2 — Versi 1 (Gabungan dalam satu paragraf setelah <br>)
+                val mergedChapterElement = paragraphs.find {
+                    it.html().contains("<strong>Sinopsis:</strong><br>") &&
+                        it.html().contains(Regex("""\d+[-–]?\d*\..+<br>"""))
+                }
```

Comment:
try using `selector[attr~=regex]` in jsoup 

https://jsoup.org/apidocs/org/jsoup/select/Selector.html